packages/account-v2/src/containers/PersonalDetailsFormWithExample/__tests__/PersonalDetailsFormWithExample.spec.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Formik } from 'formik';
 import { fireEvent, render, screen, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import { getNameDOBValidationSchema } from '../../../utils/personal-details-utils';
+import { getNameDOBValidationSchema } from '../../../utils/personalDetailsUtils';
```

Comment:
Thanks for this change. Can we add this to index.js of utils folder and just consume utils?